src/stk/common/error.cpp

```diff
@@ -38,6 +38,10 @@ namespace stk
 
         _s << " (" << file << ":" << _line << ")";
 
+    #ifdef STK_STACK_TRACE
+        get_stack_trace(_s, 1);
```

Comment:
`_s << std::endl`

Reply:
Good catch, thanks. I had put it inside the function for Linux and outside for Windows. Nonsense.